packages/flutter_fuchsia/lib/example_main.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:io';
```

Comment:
This file should probably go under `bin/` next to the `lib/` directory (https://www.dartlang.org/tools/pub/package-layout). We don't want anything from here to accidentally show up in the API docs.

Reply:
Done.